Northwind.Console/Services/NorthwindApiService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Northwind.Console.Interfaces.Services;
+using Northwind.Web.Models;
+
+namespace Northwind.Console.Services
+{
+    public class NorthwindApiService : INorthwindApiService
+    {
+        private const string CategoriesPath = "categories";
+        private const string ProductsPath = "products";
+
+        private readonly HttpClient _httpClient;
+
+        public NorthwindApiService(HttpClient httpClient)

```

Comment:
Then here, you would inject `IHttpClientFactory` instead of HttpClient. And then ask for a client like:
_httpClient = factory.CreateClient("myapiclient");